products/mlengine/api.yaml

```diff
@@ -235,4 +235,354 @@ objects:
+      ## Training output start here      
+      - !ruby/object:Api::Type::NestedObject
+        name: 'trainingOutput'
+        description: The current training job result.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'completedTrialCount'
+            description: The number of hyperparameter tuning trials that completed successfully. Only set for hyperparameter tuning jobs.
+            output: true
+          - !ruby/object:Api::Type::NestedObject
+            name: 'trials'
+            description: Results for individual Hyperparameter trials. Only set for hyperparameter tuning jobs.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'trialId'
+                description: The trial id for these results.
+                output: true
+              - !ruby/object:Api::Type::KeyValuePairs
+                name: 'hyperparameters'
+                description: |
+                  The hyperparameters given to this trial. An object containing a list of "key": value pairs. 
+                  Example: { "name": "wrench", "mass": "1.3kg", "count": "3" }
+                output: true
+              - !ruby/object:Api::Type::Time
+                name: 'startTime'
+                description: Start time for the trial.
+                output: true
+              - !ruby/object:Api::Type::Time
+                name: 'endTime'
+                description:  End time for the trial.
+                output: true              
+              - !ruby/object:Api::Type::Enum
+                name: 'state'
+                description: The detailed state of a job.
+                values:
+                  - STATE_UNSPECIFIED
+                  - QUEUED
+                  - PREPARING
+                  - RUNNING
+                  - SUCCEEDED
+                  - FAILED
+                  - CANCELLING
+                  - CANCELLED
+                output: true
+              - !ruby/object:Api::Type::NestedObject
+                name: 'finalMetric'
+                description:  The final objective metric seen for this trial.
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: 'trainingStep'
+                    description:  The global training step for this metric
+                    output: true
+                  - !ruby/object:Api::Type::Integer
+                    name: 'objectiveValue'
+                    description:  The objective value at this training step.
+                    output: true
+              - !ruby/object:Api::Type::Boolean
+                name: 'isTrialStoppedEarly'
+                description:  True if the trial is stopped early.
+                output: true
+              - !ruby/object:Api::Type::NestedObject
+                name: 'allMetrics'
+                description:  All recorded object metrics for this trial. This field is not currently populated.
```

Comment:
This is output fields

Reply:
Can you clarify please? I can see that you've marked it as output, but if, as the description says, it isn't populated, then I'm not sure why our users would want this object in their state file. That's the question I'm looking for an answer for.

Reply:
Shall, I exclude this?

Reply:
That's my inclination. In the  initial release of the resource, I'd rather have only the information that we think customers are going to use, since it's easier to add things later than it is to remove them if we discover they're unnecessary or difficult to maintain.

Reply:
Ok, I am removing it. Since it is output, only api.yaml change is needed.